evm/vm/execution_context.py

```diff
@@ -0,0 +1,62 @@
+class ExecutionContext:
```

Comment:
Is there a reason this is an old-style python object?

Reply:
```python
class AwesomeNewstyleOnlyInPython3:
    ...
```

It's only old-style in python2.  In python 3 this is equivalent to `class Thing(object)`.

Reply:
🤯